### PR TITLE
Test that messaging an user sends an email

### PR DESCRIPTION
**Pull request by pallix**

### CODE REVIEW

**Comment by pallix** [comment]
I will add one more test to this branch.

**Comment by pallix** [comment]
I'm done with the branch.

File: src/adhocracy_frontend/adhocracy_frontend/tests/acceptance/shared.js [hunk 1]
```diff
@@ -11,6 +11,13 @@ var hasClass = function (element, cls) {
     });
 };
 
+var diffArray = function(a, b) {
```
Comment (nidico) [inline src/adhocracy_frontend/adhocracy_frontend/tests/acceptance/shared.js]: Can we use `_.difference` instead?

**Comment by nidico** [comment]
Nice, LGTM!

**Pull request merged**